src/ui/pages/screen/Screen.page.tsx

```diff
@@ -0,0 +1,239 @@
+        { netid: 'kevink11', org: 'Infrastructure Committee' },
+        { netid: 'mattt92', org: 'SIGtricity' },
+        { netid: 'stephenj45', org: 'SIGAIDA' },
+        { netid: 'victorc16', org: 'GLUG' },
+        { netid: 'susana80', org: 'SIGPwny' },
+        { netid: 'patrickh37', org: 'SIGQuantum' },
+      ];
+
+      // retrieve from azure active directory (aad)
+      const userNamesResponse: UserNames = [
+        { netid: 'johnd01', firstName: 'John', lastName: 'Doe' },
+        { netid: 'miker44', firstName: 'Michael', lastName: 'Roberts' },
+        { netid: 'chrisb19', firstName: 'Christopher', lastName: 'Brown' },
+        { netid: 'ethanw12', firstName: 'Ethan', lastName: 'Wong' },
+        { netid: 'emilyh54', firstName: 'Emily', lastName: 'Hernandez' },
+        { netid: 'juliel08', firstName: 'Julie', lastName: 'Lopez' },
+        { netid: 'rachelb03', firstName: 'Rachel', lastName: 'Bell' },
+        { netid: 'ashleyc28', firstName: 'Ashley', lastName: 'Clark' },
+        { netid: 'briand77', firstName: 'Brian', lastName: 'Davis' },
+        { netid: 'meganf65', firstName: 'Megan', lastName: 'Flores' },
+        { netid: 'danielh04', firstName: 'Daniel', lastName: 'Hughes' },
+        { netid: 'lindam29', firstName: 'Linda', lastName: 'Martinez' },
+        { netid: 'paulf31', firstName: 'Paul', lastName: 'Fisher' },
+        { netid: 'markl13', firstName: 'Mark', lastName: 'Lewis' },
+        { netid: 'carolynb59', firstName: 'Carolyn', lastName: 'Barnes' },
+        { netid: 'nataliep71', firstName: 'Natalie', lastName: 'Price' },
+
+        { netid: 'ethanc12', firstName: 'Ethan', middleName: 'Yuting', lastName: 'Chang' },
+        { netid: 'sarahg23', firstName: 'Sarah', middleName: 'Grace', lastName: 'Gonzalez' },
+        { netid: 'annaw02', firstName: 'Anna', middleName: 'Marie', lastName: 'Williams' },
+        { netid: 'laurenp87', firstName: 'Lauren', middleName: 'Patricia', lastName: 'Perez' },
+        { netid: 'kevink11', firstName: 'Kevin', middleName: 'Lee', lastName: 'Kim' },
+        { netid: 'mattt92', firstName: 'Matthew', middleName: 'Thomas', lastName: 'Taylor' },
+        { netid: 'stephenj45', firstName: 'Stephen', middleName: 'James', lastName: 'Johnson' },
+        { netid: 'victorc16', firstName: 'Victor', middleName: 'Charles', lastName: 'Carter' },
+        { netid: 'susana80', firstName: 'Susan', middleName: 'Ann', lastName: 'Anderson' },
+        { netid: 'patrickh37', firstName: 'Patrick', middleName: 'Henry', lastName: 'Hill' },
+      ];
+
+      const mergedResponse: Users = userOrgsResponse.map((orgObj) => {
+        const nameObj = userNamesResponse.find((name) => name.netid === orgObj.netid);
+        return { ...orgObj, ...nameObj } as User;
+      });
+
+      setUserList(mergedResponse);
+    };
+    getUsers();
+  }, []);
+
+  const removeUser = async (netid: string) => {
+    try {
+      // await api.delete(`/api/v1/events/${eventId}`);
```

Comment:
we would probably call something like `PATCH /api/v1/iam/:orgName/leads` with a body {remove: [email]}